lib/index.js

```diff
@@ -462,6 +464,31 @@ exports.timesSeries = function(times, task) {
 	});
 };
 
+exports.memoize = function(promiseFn, hasher) {
+	var callbackFn = constructIterator(promiseFn, 1, true, false);
```

Comment:
Does the function passed to memoize() support numbers of arguments other than 1?